examples/unilm/README.md

```diff
@@ -0,0 +1,57 @@
+# Unilm
+
+- Paper
+	- [UnilmV1](https://arxiv.org/abs/1905.03197)
+	- [UnilmV2](https://arxiv.org/abs/2002.12804)
+- [Open Source](https://github.com/microsoft/unilm)
+
+## Speedup by using FastSeq
+
+- CNN daily mail validation data, NVIDIA-2080TI
+
+  |       BatchSize      |       32      |        64       |      128       |
+  |:--------------------:|:-------------:|:---------------:|:--------------:|
+  |      unilm           |  - samples/s  |  1.2 samples/s  |      OOM       |
+  |   above + fastseq    |  - samples/s  |  8.4 samples/s  |  8.4 samples/s |
```

Comment:
any idea why bs32 result is empty? bs128 speed is same as bs64, looks strange.

Reply:
This result is from my local machine with 2080TI, which only have 11G memory. Both of the GPU usage with bs=64 and 128 is almost 100%. The difference of speed is very a little. This results is comparison of unilm offical repo with import fastseq or not.

Reply:
In addition, unilm offical repo has its own implementation of beam search, which is a little different from the one in huggingface. Which one is better as a baseline for unilm ? Unilm official repo or optimized-unilm in fastseq with GenerationMixin ?

Reply:
Should be the one from unilm official repo?

Reply:
I think so. This table could be updated after running it with V100~ This may need the help of @feihugis .

Reply:
it is ok. let's update it in separate pr.

Reply:
Synced with @fuliucansheng offline. The numbers in hf_unilm.sh are copied from the template file. I will run the benchmarks and update the benchmark numbers in another PR.